client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -141,10 +107,18 @@ class CustomContentTypes extends Component {
 		const { translate } = this.props;
 		return (
 			<div>
-				<SectionHeader label={ translate( 'Custom Content Types' ) } />
+				<SectionHeader label={ translate( 'Custom content types' ) } />
 
 				<Card className="custom-content-types__card site-settings">
 					<FormFieldset>
+						<div className="custom-content-types__info-link-container site-settings__info-link-container">
+							<InfoPopover position={ 'left' }>
+								<ExternalLink href={ 'https://support.wordpress.com/custom-post-types/' } icon target="_blank">
+									{ translate( 'Learn more about Custom Content Types.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Learn more about Custom Content Types' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).